dev-requirements.txt

```diff
@@ -1,9 +1,7 @@
-black==19.10b0 ; python_version >= '3.6'
+black==19.10b0
 pytest<6
 pytest-qt
-mock ; python_version >= '3'
```

Comment:
should we not still keep mock?

Reply:
https://github.com/equinor/ert/pull/1010#discussion_r485591209

We're using `unittest.mock` which is a part of CPython 3